### ISSUE TITLE
New API Update

### ISSUE THREAD

**Issue by PoGoWeb** [tuttarealstep/PokeGOAPI-PHP#14]
Hi, are you going to be upgrading this to work with the latest API at all? Thanks.

**Comment by MajorChump** [comment]
@tuttarealstep Looks like they've added support over at https://github.com/Grover-c13/PokeGOAPI-Java for unknown6, is this something you're working on?

**Comment by tuttarealstep** [comment]
Yes now I'm upgrading but I have some problem with the time in this week.

**Comment by MajorChump** [comment]
I've handled the xxhash dependency here to add support for seeds. https://github.com/MatthewKingDev/php-xxhash should make it easier to complete
